strings/ion_laws.json

```diff
@@ -1,5 +1,132 @@
 {
-    "ionabstract": [
+    "ionspecies": [
+        "LIFEFORMS WITHOUT TAILS",
+		"LIFEFORMS WITHOUT WINGS",
+		"INORGANIC LIFEFORMS",
+		"LIFEFORMS WITH TAILS",
+		"LIFEFORMS WITH WINGS",
+		"LIFEFORMS HOLDING A SOURCE OF LIGHT",
+		"LIFEFORMS HOLDING A TOOLBOX",
+		"LIFEFORMS HOLDING A FIRE EXTINGUISHER",
+		"LIFEFORMS WITH AN OBSCURED IDENTITY",
+		"LIFEFORMS THAT HAVE FARTED RECENTLY",
+		"LIFEFORMS THAT ARE NOT WEARING A BAG",
+		"LIFEFORMS THAT ARE WEARING A PAPER HAT",
+		"LIFEFORMS THAT ARE WEARING A CLOAK",
+		"LIFEFORMS THAT ARE NOT WEARING SHOES",
+		"LIFEFORMS THAT HAVE RECENTLY SAID NYA"
+    ],
+    "ionfragment": [
+        "VISIBLY CARRIES A GUN",
+		"HAS BLOOD ON THEIR CLOTHING",
+		"BREAKS SPACE LAW",
+		"HAS ANY DISEASE",
+		"IS WEARING A HARDSUIT"
+    ],
+    "iondepartment": [
+        "SECURITY",
+		"ENGINEERING",
+		"THE MEDBAY",
+		"RESEARCH AND DEVELOPMENT",
+		"MAINTENANCE",
+		"BOTANY",
+		"CARGO",
+		"SPACE"
```

Comment:
Suggestion: Lavaland

Reply:
"mining outpost" would work better, puts the AI in a powered area.

Reply:
Can do